apps/organizations/models.py

```diff
@@ -0,0 +1,91 @@
+"""
+Declare and configure the models related to Organizations
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from cms.extensions import PageExtension
+from cms.extensions.extension_pool import extension_pool
+
+ORGANIZATIONS_PAGE_REVERSE_ID = 'organizations'
+
+
+class Organization(models.Model):
+    """
+    The Organization model is used to represent and record entities that manage courses.
+    It could be a university or a training company for example.
+
+    This model should be used to record structured data about the organization whereas the
+    associated page object is where we record the less structured information to display on the
+    page to present the organization.
+    """
```

Comment:
Why not giving this model a [`UUIDField`](https://docs.djangoproject.com/en/2.0/ref/models/fields/#uuidfield) as primary key? (I'm a big fan of uuids).

Reply:
I'm pro uuidfield too.